src/components/OrganizationSaas/Dashboard/EditProfile.tsx

```diff
@@ -0,0 +1,243 @@
+import {
+  AspectRatio,
+  Box,
+  Flex,
+  FormControl,
+  FormLabel,
+  IconButton,
+  Image,
+  Input,
+  Text,
+  Textarea,
+} from '@chakra-ui/react'
+import { Button } from '@vocdoni/chakra-components'
+import { errorToString, useClient, useOrganization } from '@vocdoni/react-providers'
+import { Account } from '@vocdoni/sdk'
+import { useState } from 'react'
+import { FormProvider, SubmitHandler, useForm } from 'react-hook-form'
+import { Trans, useTranslation } from 'react-i18next'
+import { BiTrash } from 'react-icons/bi'
+import { BsFillTrashFill } from 'react-icons/bs'
+import { MdBrowserUpdated } from 'react-icons/md'
+import InputCustom from '~components/Layout/InputCustom'
+import {
+  CountriesTypesSelector,
+  CustomizationLanguageSelector,
+  CustomizationTimeZoneSelector,
+  MembershipSizeTypesSelector,
+  OrganzationTypesSelector,
+} from '~components/Layout/SaasSelector'
+import useDarkMode from '~src/themes/saas/hooks/useDarkMode'
+import fallback from '/assets/default-avatar.png'
+
+interface EditFormFields {
+  name: string
+  description: string
+  avatar: string
+}
+
+const REGEX_AVATAR = /^(https?:\/\/|ipfs:\/\/)/i
+
+const EditProfile = () => {
+  const { textColor, textColorSecondary, bgSecondary } = useDarkMode()
+  const { account, fetchAccount } = useClient()
+  const { update } = useOrganization()
+  const { t } = useTranslation()
+  const [loading, setLoading] = useState(false)
+  const [error, setError] = useState<string | null>(null)
+
+  const required = {
+    value: true,
+    message: t('form.error.field_is_required'),
+  }
+
+  const methods = useForm({
+    defaultValues: {
+      name: account?.account.name.default || '',
+      description: account?.account.description.default || '',
+      avatar: account?.account.avatar || '',
+    },
+  })
+
+  const {
+    watch,
+    setValue,
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = methods
+
+  const avatar = watch('avatar')
+
+  const correctAvatarFormat = (val: string) => REGEX_AVATAR.test(val)
+
+  const onSubmit: SubmitHandler<EditFormFields> = async (values: EditFormFields) => {
+    setLoading(true)
+
+    try {
+      await update(new Account({ ...account?.account, ...values }))
+      fetchAccount()
+    } catch (err: any) {
+      setError(errorToString(err))
+    } finally {
+      setLoading(false)
+    }
+  }
+
+  return (
+    <FormProvider {...methods}>
+      <Box height='100%' maxH='100%' overflowY='auto'>
+        <Flex as='form' flexDirection='column' gap={12} maxW='600px' mx='auto'>
+          <Flex flexDirection='column' gap={6}>
+            <Box>
+              <Text color={textColor} fontWeight='bold'>
+                {t('edit.public_org_info', { defaultValue: 'Public Organization Information' })}
```

Comment:
On all the file you are using `edit` as "main" i18nKey. Imo this is not good: `edit` is too general, can be a unique word, could refer to many points on the code etc... 

Instead, replace it (on the whole file) by `edit_saas_profile` or similar, which won't cause collisions to any other translations keys

Reply:
3727e1a